back/src/adapters/primary/routers/address/addressRouter.e2e.test.ts

```diff
@@ -17,101 +19,98 @@ const lookupAddressQueryParam = "lookup";
 const lookupLocationQueryParam = "query";
 
 const lookupStreetAddressUrl = (lookup: LookupAddress): string =>
-  `${addressTargets.lookupStreetAddress.url}?${lookupAddressQueryParam}=${lookup}`;
+  `${addressRoutes.lookupStreetAddress.url}?${lookupAddressQueryParam}=${lookup}`;
 
 const lookupLocationUrl = (lookupLocationInput: LookupLocationInput): string =>
-  `${addressTargets.lookupLocation.url}?${lookupLocationQueryParam}=${lookupLocationInput}`;
+  `${addressRoutes.lookupLocation.url}?${lookupLocationQueryParam}=${lookupLocationInput}`;
 
 describe("addressRouter", () => {
-  let request: SuperTest<Test>;
+  let httpClient: HttpClient<AddressRoutes>;
   let addressGateway: InMemoryAddressGateway;
 
   beforeEach(async () => {
-    const testAppAndDeps = await buildTestApp();
-    request = testAppAndDeps.request;
-    addressGateway = testAppAndDeps.gateways.addressApi;
+    const { request, gateways } = await buildTestApp();
+    httpClient = createSupertestSharedClient(addressRoutes, request);
+    addressGateway = gateways.addressApi;
   });
 
-  describe(`${addressTargets.lookupStreetAddress.url} route`, () => {
+  describe(`${addressRoutes.lookupStreetAddress.url} route`, () => {
```

Comment:
si on veut il y a la petite fonction pour avoir la method et l'url :
```suggestion
  describe(`${displayRouteName(addressRoutes.lookupStreetAddress)} to get street addresses`, () => {
```

Reply:
sympa je l'avais pas vu cette méthode